python/lsst/ts/observatory/control/auxtel/atcs.py

```diff
@@ -508,16 +534,32 @@ async def shutdown(self):
         move the telescope and dome to the park position and disable all
         components.
         """
+        # Create a copy of check to restore at the end.
+        check = copy.copy(self.check)
+
         self.log.info("Disabling ATAOS corrections")
 
         try:
 
             await self.rem.ataos.cmd_disableCorrection.set_start(
                 disableAll=True, timeout=self.long_timeout
             )
-        except Exception as e:
-            self.log.warning("Failed to disable ATAOS corrections. Continuing...")
-            self.log.exception(e)
+        except Exception:
+            self.log.exception("Failed to disable ATAOS corrections. Continuing...")
```

Comment:
Are you sure you don't want to log the error that was caught anymore? It may be useful when debugging problems. Possibly review all cases like this in this file.

Reply:
It is a neat trick that I learned from Russell... When you use `log.exception` it also adds the traceback at the end.. :)

```
In [1]: import logging

In [2]: try:
   ...:     raise RuntimeError("This is a mistake.")
   ...: except Exception:
   ...:     logging.exception("oh no!")
   ...:
ERROR:root:oh no!
Traceback (most recent call last):
  File "<ipython-input-2-e51e6cc87a80>", line 2, in <module>
    raise RuntimeError("This is a mistake.")
RuntimeError: This is a mistake.
```

Reply:
Thanks for teaching me that trick!